<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>specNormalizeActionDialog</class>
 <widget class="QDialog" name="specNormalizeActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QRadioButton" name="minimum">
     <property name="text">
      <string>Minimum</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="maximum">
     <property name="text">
      <string>Maximum</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="shiftXValue">
       <property name="text">
        <string>Shift x value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="xValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="scaleYValue">
       <property name="text">
        <string>Scale y value</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="yValue">
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>66</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>minimum</tabstop>
  <tabstop>maximum</tabstop>
  <tabstop>shiftXValue</tabstop>
  <tabstop>xValue</tabstop>
  <tabstop>scaleYValue</tabstop>
  <tabstop>yValue</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>specNormalizeActionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>specNormalizeActionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shiftXValue</sender>
   <signal>toggled(bool)</signal>
   <receiver>xValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaleYValue</sender>
   <signal>toggled(bool)</signal>
   <receiver>yValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
